packages/core/src/types/index.ts

```diff
@@ -354,6 +355,8 @@ export interface DropdownStyleProps {
 export interface NestedInputProps extends BaseFormikFieldsWithHandleChange {
   buttonText: string;
   placeholder: string;
+  preventDefault: boolean;
+  handleClick: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Believe this should be optional, since in other cases we may not want to pass a handleClick function